examples/java/spring-boot/src/main/java/uk/gov/api/springboot/dtos/Api.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.api.springboot.dtos;
+
+/** Domain object for API (metadata) objects. */
+public record Api(
```

Comment:
Not sure this is meant to still be here :thinking: I think this is to be deleted when we have our commit to migrate to Onion